Monika After Story/game/chess.rpy

```diff
@@ -958,7 +959,22 @@ label mas_chess_remenu:
 
         #Normal/Really Bad Chess/Chess 960 selection
         if menu_category == "gamemode_select":
-            $ chessmode = _return
+            if _return is 0:
```

Comment:
Don't use `is` with literals. Also probably make it a string constant like the rest, for example, `"explain_rules"`.